core/templates/dev/head/galleries/Gallery.js

```diff
@@ -203,6 +203,51 @@ oppia.controller('Gallery', [
       $rootScope.loadingMessage = '';
     });
 
+    $scope.leftCheck = function(ind) {
+      var tilesHtml = '.oppia-gallery-tiles:eq(n)'.replace('n', ind);
+
+      // Checks if it is at the left most point.
+      return parseInt($(tilesHtml).css('left')) >= 0;
+    };
+
+    $scope.scrollLeft = function(ind) {
+      if (!$scope.leftCheck(ind)) {
+        var tilesHtml = '.oppia-gallery-tiles:eq(n)'.replace('n', ind);
+        $(tilesHtml).animate({
+          left: '+=211.953'
+        }, 500);
+      }
+    };
+
+    $scope.rightLowerLimits = [];
+    $scope.carouselDisplayWidth = $('.oppia-gallery-tiles-carousel').width();
+    $scope.rightCheck = function(ind, galleryGroup) {
+      var leftLowerLimit = $scope.rightLowerLimits[ind];
```

Comment:
I also don't understand the terminology -- can you find a better name to convey the concept? I don't understand what 'leftLowerLimit' and 'rightLowerLimit' are.